packages/proxy-container/package.json

```
@@ -0,0 +1,75 @@
+{
+	"name": "@discordjs/proxy-container",
+	"version": "0.1.0-dev",
+	"description": "Lightweight HTTP proxy for Discord's API, brought to you as a container 📦",
+	"scripts": {
+		"build": "tsup && tsc --noEmit --incremental",
+		"lint": "prettier --check . && eslint src --ext mjs,js,ts",
+		"format": "prettier --write . && eslint src --ext mjs,js,ts --fix",
+		"prepublishOnly": "yarn build && yarn lint && yarn test",
+		"changelog": "git cliff --prepend ./CHANGELOG.md -u -c ./cliff.toml -r ../../ --include-path 'packages/proxy-container/*'"
+	},
+	"main": "./dist/index.js",
+	"module": "./dist/index.mjs",
+	"typings": "./dist/index.d.ts",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js",
+		"types": "./dist/index.d.ts"
+	},
```

No exports, only `main` is to be kept, also missing the type to run the ESM code:
```suggestion
	"type": "module",
```